server/src/main/java/io/spine/server/aggregate/AggregateRepository.java

```diff
@@ -116,21 +117,32 @@ protected AggregateRepository() {
      * {@inheritDoc}
      *
      * <p>{@linkplain io.spine.server.commandbus.CommandBus#register(
-     *io.spine.server.bus.MessageDispatcher) Registers} itself with the {@code CommandBus} of the
+     * io.spine.server.bus.MessageDispatcher) Registers} itself with the {@code CommandBus} of the
      * parent {@code BoundedContext}.
      */
     @Override
     public void onRegistered() {
         super.onRegistered();
         final BoundedContext boundedContext = getBoundedContext();
-        boundedContext.getCommandBus()
-                      .register(this);
 
-        // If there are any events on which aggregates react, register via delegating dispatcher.
+        final Set<CommandClass> commandClasses = getMessageClasses();
+
         final DelegatingEventDispatcher<I> delegatingDispatcher;
         delegatingDispatcher = DelegatingEventDispatcher.of(this);
-        if (!delegatingDispatcher.getMessageClasses()
-                                 .isEmpty()) {
+        final Set<EventClass> eventClasses = delegatingDispatcher.getMessageClasses();
+
+        if (commandClasses.isEmpty() && eventClasses.isEmpty()) {
+            throw newIllegalStateException(
```

Comment:
As in the previous PRs this branch is reported to be uncovered. However, as before the suspects are usual (i.e. Codecov).